.ci/env/apt.sh

```diff
@@ -34,8 +34,13 @@ function install_dpcpp {
     sudo apt-get install -y intel-oneapi-compiler-dpcpp-cpp-2025.0 intel-oneapi-runtime-libs=2025.0.0-406
 }
 
+function install_tbb {
+    sudo apt-get install -y intel-oneapi-tbb-devel-2022.0
+}
+
 function install_mkl {
-    sudo apt-get install -y intel-oneapi-mkl-devel-2025.0 intel-oneapi-tbb-devel-2022.0
+    sudo apt-get install -y intel-oneapi-mkl-devel-2025.0
+    install_tbb
```

Comment:
should it be called here? May be its better to call both functions? And why we must include it in MKL?

Reply:
Good question @Alexandr-Solovev, why tbb install was included in mkl I do not know (included in the 2025.0 update I think), but I called one function in the other so that we only have to change the tbb version in one location going forward. Let me know if I should change it.